src/server_manager/ui_components/outline-server-settings.html

```diff
@@ -208,6 +245,77 @@ <h3>[[localize('settings-metrics-header')]]</h3>
           this.fire(metricsSignal);
         }
       },
+      _toggleKeysPortButtonsVisible: function(visible) {
```

Comment:
I think "toggle" taking the state is an uncommon pattern. Maybe use `setKeysPortButtonsVisible` instead?

Reply:
Done